docs/getting-started/data-views-in-sec.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[data-views-in-sec]]
+= {Data-sources} in Elastic Security
+
+The active {data-source} determines which data appears throughout {es-sec}.
+{Data-sources} are defined by the index patterns they include.
+Only data from the {es} {ref}/documents-indices.html[indices], {ref}/data-streams.html[data streams], or {ref}/alias.html[index aliases] specified in the active {data-source} will appear.
+
+
+== How to switch {data-sources}
+
+You can tell which {data-source} is active by clicking the {Data-source} button that appears at the upper right of {es-sec} pages that display data, such as Overview, Alerts, or Hosts.
+To switch to another existing {data-source}, click the same button {Data-source} and choose a different {data-source} under `Choose {data-source}`:
+
+image::images/dataview-default-setting-highlighted.png[the securitySolution:defaultIndex setting, shown in context]
```

Comment:
Think this image got accidentally switched around with the image in the "The default data view" section. :)